test/unit/test-amp-story-player.js

```diff
@@ -370,5 +370,49 @@ describes.realWin('AmpStoryPlayer', {amp: false}, (env) => {
         'Story URL not found in the player: https://example.com/story6.html'
       );
     });
+
+    it('navigate to stories given a number', async () => {
+      const playerEl = win.document.createElement('amp-story-player');
+      appendStoriesToPlayer(playerEl, 5);
+
+      const player = new AmpStoryPlayer(win, playerEl);
+
+      await player.load();
+
+      const stories = toArray(playerEl.querySelectorAll('a'));
+
+      player.go(0);
+      expect(stories[0][IFRAME_IDX]).to.eql(0);
+      player.go(1);
+      expect(stories[0][IFRAME_IDX]).to.eql(1);
+      player.go(-1);
+      expect(stories[0][IFRAME_IDX]).to.eql(0);
```

Comment:
This test does not currently pass, I am unsure how to check the story's iframe index property.

Reply:
So what `IFRAME_IDX` tells you is the iframe index that corresponds to a given story (if any), but this doesn't really tell you which of those is the currently active iframe. 

I guess a more appropriate check here would be to check the iframe position, like we do in other tests:

https://github.com/ampproject/amphtml/blob/33e0c090956904a9adcf1d877b4c7727a6352543/test/unit/test-amp-story-player.js#L219

Reply:
When iframe position is `0`, it is the currently displayed story, when it is either `1` or `-1` it means the iframe is at either side of the story.

Reply:
I just changed the test, it still currently fails. But let know if I'm on the right track please.